.github/workflows/release-plz.yml

```diff
@@ -0,0 +1,28 @@
+name: Release Plz
+
+permissions:
+  pull-requests: write
+  contents: write
+
+on:
+  push:
+    branches:
+      - dev
+      # - main
```

Comment:
I will change this to `main` after but I just want to see it make a release PR for `dev` first to test it out